assets/js/lib/test-utils/factories/advisoryErrata.js

```diff
@@ -31,6 +39,8 @@ export const advisoryErrataFactory = Factory.define(({ params }) => ({
       { transient: { length: faker.number.int({ min: 1, max: 10 }) } }
     ),
   cves: cveFactory.buildList(10),
+  affected_packages:
+    params.affected_packages || affectedPackageFactory.buildList(10),
```

Comment:
```suggestion
    affectedPackageFactory.buildList(10),
```

If you use an empty array in tests this will be replaced automatically without the need of this `or` expression :+1:

Reply:
Omg yeah it's supposed to be an array anyway... I think the time got to me. Thanks for pointing this out ^^